SCSS/Imports/_museum.scss

```diff
@@ -42,3 +42,18 @@
   color: #ffffff;
 }
 
+.pagination{
+  .btn {
+    margin: 5px;
```

Comment:
not sure this is working? maybe try adding a class of m-1 to the buttons and make use of bootstrap :)